mobile/src/components/common/icon/icon.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+
+import { IconName } from '~/common/types/ui/icon-name.type';
+import { IconNameToIcon } from '~/components/common/icon/common/maps/icon-name-to-icon.map';
```

Comment:
Re-export it from maps.ts
```suggestion
import { IconNameToIcon } from '../maps/maps';
```